src/rebilly-js-sdk.js

```diff
@@ -23,11 +23,13 @@ const baseTimeoutMs = 6000;
  * Create an instance of the Rebilly API
  * @param {ApiParams} 
  */
-export default function RebillyAPI({apiKey = null, sandbox = false, timeout = baseTimeoutMs, organizationId = null, urls = baseEndpoints} = {}) {
-    if(!urls.live || !urls.sandbox) {
+export default function RebillyAPI( // This default export is kept for backwards compatibility (better use RebillyAPI named export when consuming importing library)
+    {apiKey = null, sandbox = false, timeout = baseTimeoutMs, organizationId = null, urls = baseEndpoints} = {}
```

Comment:
```suggestion
// This default export is kept for backwards compatibility (better use RebillyAPI named export when consuming importing library)
export default function RebillyAPI({
    apiKey = null, sandbox = false, timeout = baseTimeoutMs, organizationId = null, urls = baseEndpoints
} = {}
```

With the bracket in the comment it wasn't clear what the shape of the function was.